runtime/integration-tests/src/pendulum_tests.rs

```diff
@@ -116,3 +129,16 @@ fn assethub_transfer_asset_to_pendulum_and_back() {
 		network_id
 	);
 }
+
+#[test]
+fn transfer_native_token_to_sibling_parachain_and_back() {
```

Comment:
Nit: let's use explicit parachain names here so that it's easier to distinguish the tests of pendulum and amplitude. I would suggest we use `transfer_native_token_from_pendulum_to_sibling_parachain_and_back()`